### DISCUSSION TITLE
Unity on Flutter

### DISCUSSION THREAD

**Discussion by braysonjohn148** [ant-media/Ant-Media-Server#6362] [General]
I am building a Flutter App, and I want to implement Face Filters, so I went for Unity. How can I stream my Unity Widget Screen to Antmedia servers?

**Comment by zerostack0** [comment]
Hi @braysonjohn148 

I didn't test embedding Unity into Flutter before, can you explain how it works?

In normal circumstances, all you need to do is create a custom Media Stream from the video which has filtered video and set it using the following function https://github.com/ant-media/WebRTC-Flutter-SDK/blob/main/lib/src/helpers/helper.dart#L468

If unity works as iFrame and there is no communication between unity and flutter, then you can take a look at unity SDK directly.
